addon-test-support/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Used to set up engine resolver for a unit test.
```

Comment:
These docs are a bit wrong now (given the new implementation)

Reply:
fixed!